docs/enterprise-setup/upgrade-service-account.md

```diff
@@ -0,0 +1,66 @@
+---
+products: oss-enterprise
+---
+
+# Update service account for 1.6
```

Comment:
Can we move this to the SME section please? Since this doesn't impact 99% of users?